src/order_item_summary/OrderItemSummaryController.ts

```diff
@@ -20,15 +21,15 @@ export class OrderItemSummaryController {
         const signInInfo = request.session?.data[SessionKey.SignInInfo];
         const apiToken = signInInfo?.[SignInInfoKeys.AccessToken]?.[SignInInfoKeys.AccessToken]!;
         const userId = signInInfo?.[SignInInfoKeys.UserProfile]?.[UserProfileKeys.UserId];
+        const pageHeader = mapPageHeader(request)
         try {
             logger.debug(`Retrieving summary for order/item [${orderId}/${itemId}] for user [${userId}]...`);
             const viewModel = await this.service.getOrderItem({
                 orderId,
                 itemId,
                 apiToken
             });
-            logger.debug(`Retrieved summary for order/item [${orderId}/${itemId}] for user [${userId}]`);
```

Comment:
I think this logging statement was there from before. It may be generally useful?